### PR TITLE
[kubernetes/hdfs] make NetworkPolicy installation optional

### LINKED ISSUE #156: [kubernetes/hdfs] Makes NetworkPolicy optional
## Feature Request
In some cases, people may want to disable `NetworkPolicy` installation. Make NetworkPolicy installation optional

### PR DESCRIPTION
**Pull request by ahmetkaftan**
closes #156 
Signed-off-by: Ahmet Kaftan <ahmet.kaftan@cloudnesil.com>

### CODE REVIEW

**Review by t92549 — CHANGES_REQUESTED** [review]

**Comment by t92549** [comment]
Would it be more useful to offer a way to change the network policy instead of disabling it entirely? For example, you could offer a way to change the `podSelector` to match more pods or even whitelist some IPs using an `ipBlock`. This would mean the hdfs nodes are still secure but they can be altered to match application needs.

File: kubernetes/hdfs/Chart.yaml [hunk 1]
```diff
@@ -16,5 +16,5 @@ apiVersion: v2
 name: hdfs
 description: Deploys the Hadoop Distributed File System (HDFS)
 type: application
-version: 0.10.0 # managed version
+version: 0.10.1 # managed version
```
[the remark this answers is not in the document]
Comment (ahmetkaftan) [reply kubernetes/hdfs/Chart.yaml]: Reverted version bump

File: kubernetes/hdfs/values.yaml [hunk 2]
```diff
@@ -120,3 +120,7 @@ postInstallCommands: []
 # - hadoop fs -chown accumulo /accumulo
 # - hadoop fs -chmod 700 /accumulo
 # - hadoop fs -ls /
+
+networkPolicy:
+  ## Specifies whether a NetworkPolicies should be created
```
[the remark this answers is not in the document]
Comment (ahmetkaftan) [reply kubernetes/hdfs/values.yaml]: Removed comment.

**Comment by ahmetkaftan** [comment]
> Would it be more useful to offer a way to change the network policy instead of disabling it entirely? For example, you could offer a way to change the `podSelector` to match more pods or even whitelist some IPs using an `ipBlock`. This would mean the hdfs nodes are still secure but they can be altered to match application needs.

Yes, that's would be great but what if I or someone else handle this as a separate issue?

**Review by t92549 — APPROVED** [review]

**Review by t92549 — APPROVED** [review]

**Comment by t92549** [comment]
> > Would it be more useful to offer a way to change the network policy instead of disabling it entirely? For example, you could offer a way to change the `podSelector` to match more pods or even whitelist some IPs using an `ipBlock`. This would mean the hdfs nodes are still secure but they can be altered to match application needs.
> 
> Yes, that's would be great but what if I or someone else handle this as a separate issue?

Okay well we will merge this now but yes it would be nice to make this into another issue.

**Pull request merged**